2025-03-22 11:14:13 - 99 - launch_app - [ERROR] - Error: The script 'C:\ExampleFolder\ExampleScript.py' does not exist.
2025-03-22 11:14:14 - 99 - launch_app - [ERROR] - Error: The script 'C:\ExampleFolder\ExampleScript.py' does not exist.
2025-03-22 11:14:14 - 99 - launch_app - [ERROR] - Error: The script 'C:\ExampleFolder\ExampleScript.py' does not exist.
2025-03-22 11:14:59 - 74 - launch_app - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-22 11:14:59 - 75 - launch_app - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-22 11:14:59 - 94 - launch_app - [INFO] - Successfully launched the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-22 11:15:02 - 74 - launch_app - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-22 11:15:02 - 75 - launch_app - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-22 11:15:02 - 94 - launch_app - [INFO] - Successfully launched the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-22 11:15:03 - 74 - launch_app - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-22 11:15:03 - 75 - launch_app - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-22 11:15:03 - 94 - launch_app - [INFO] - Successfully launched the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-22 11:15:03 - 74 - launch_app - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-22 11:15:03 - 75 - launch_app - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-22 11:15:03 - 94 - launch_app - [INFO] - Successfully launched the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-22 11:15:03 - 74 - launch_app - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-22 11:15:03 - 75 - launch_app - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-22 11:15:03 - 94 - launch_app - [INFO] - Successfully launched the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-22 11:17:17 - 74 - launch_app - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-22 11:17:17 - 75 - launch_app - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-22 11:17:17 - 94 - launch_app - [INFO] - Successfully launched the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-22 11:23:14 - 74 - launch_app - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-22 11:23:14 - 75 - launch_app - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-22 11:23:14 - 94 - launch_app - [INFO] - Successfully launched the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-22 11:23:42 - 74 - launch_app - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-22 11:23:42 - 75 - launch_app - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-22 11:23:44 - 94 - launch_app - [INFO] - Successfully launched the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-22 11:24:09 - 74 - launch_app - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-22 11:24:09 - 75 - launch_app - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-22 11:24:22 - 94 - launch_app - [INFO] - Successfully launched the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-22 11:41:27 - 74 - launch_app - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-22 11:41:27 - 75 - launch_app - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-22 11:43:16 - 74 - launch_app - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-22 11:43:16 - 75 - launch_app - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-22 11:43:20 - 74 - launch_app - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-22 11:43:20 - 75 - launch_app - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-28 23:08:01 - 74 - launch_app - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-28 23:08:01 - 75 - launch_app - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-28 23:08:03 - 74 - launch_app - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-28 23:08:03 - 75 - launch_app - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-28 23:08:08 - 172 - check_dict - [ERROR] - Error: Command dictionary does not contain 'path' key.
2025-03-28 23:08:09 - 172 - check_dict - [ERROR] - Error: Command dictionary does not contain 'path' key.
2025-03-28 23:08:14 - 94 - launch_app - [ERROR] - Error: The script 'C:\ExampleFolder\ExampleScript.py' does not exist.
2025-03-28 23:08:15 - 94 - launch_app - [ERROR] - Error: The script 'C:\ExampleFolder\ExampleScript.py' does not exist.
2025-03-28 23:08:16 - 74 - launch_app - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-28 23:08:16 - 75 - launch_app - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-28 23:08:33 - 94 - launch_app - [ERROR] - Error: The script 'C:\ExampleFolder\ExampleScript.py' does not exist.
2025-03-28 23:08:35 - 172 - check_dict - [ERROR] - Error: Command dictionary does not contain 'path' key.
2025-03-28 23:08:35 - 172 - check_dict - [ERROR] - Error: Command dictionary does not contain 'path' key.
2025-03-28 23:08:37 - 74 - launch_app - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-28 23:08:37 - 75 - launch_app - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-28 23:13:32 - 70 - run_script - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-28 23:13:32 - 71 - run_script - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-28 23:27:06 - 70 - run_script - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-28 23:27:06 - 71 - run_script - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-28 23:27:08 - 70 - run_script - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-28 23:27:08 - 71 - run_script - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-28 23:37:07 - 70 - run_script - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-28 23:37:07 - 71 - run_script - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-28 23:37:08 - 115 - stop_script - [INFO] - Terminating the process for button: Stop Script
2025-03-28 23:37:11 - 70 - run_script - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-28 23:37:11 - 71 - run_script - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test2.py
2025-03-28 23:37:12 - 115 - stop_script - [INFO] - Terminating the process for button: Stop Script
2025-03-28 23:37:14 - 70 - run_script - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-28 23:37:14 - 71 - run_script - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-28 23:37:16 - 92 - run_script - [ERROR] - Error: The script 'C:\ExampleFolder\ExampleScript.py' does not exist.
2025-03-28 23:37:21 - 92 - run_script - [ERROR] - Error: The script 'C:\ExampleFolder\ExampleScript.py' does not exist.
2025-03-28 23:37:22 - 92 - run_script - [ERROR] - Error: The script 'C:\ExampleFolder\ExampleScript.py' does not exist.
2025-03-28 23:37:25 - 70 - run_script - [DEBUG] - Launching the application: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-28 23:37:25 - 71 - run_script - [DEBUG] - Command: C:\Users\nicol\OneDrive\Documents\Github-Projects\Python_Script_Launcher\tests\test.py
2025-03-28 23:37:27 - 92 - run_script - [ERROR] - Error: The script 'C:\ExampleFolder\ExampleScript.py' does not exist.
2025-03-28 23:37:28 - 194 - check_dict - [ERROR] - Error: Command dictionary does not contain 'path' key.
2025-03-28 23:37:30 - 194 - check_dict - [ERROR] - Error: Command dictionary does not contain 'path' key.
2025-03-28 23:37:31 - 92 - run_script - [ERROR] - Error: The script 'C:\ExampleFolder\ExampleScript.py' does not exist.
